### PR TITLE
removing inline sql.

### PR DESCRIPTION
**Pull request by vitaly-t**
Started moving SQL outside.  There can be plenty of improvements added after this one is merged.

### CODE REVIEW

**Comment by vitaly-t** [comment]
Build fails because of indentation in file sql/index.js. How can I fix it in WebStorm?

File: src/Adapters/Storage/Postgres/sql/index.js [hunk 1]
```diff
@@ -25,7 +25,8 @@ function sql(file) {
   var qf = new QueryFile(fullPath, {minify: true});
 
   if (qf.error) {
-    console.error(qf.error);
+    // commented out because Lint doesn't permit console messages:
+    // console.error(qf.error);
```
Comment (flovilmart) [inline src/Adapters/Storage/Postgres/sql/index.js]: if we can't parse a query file, we should probably throw and exit :)

Comment (flovilmart) [inline src/Adapters/Storage/Postgres/sql/index.js]: Otherwise that looks great!

**Comment by vitaly-t** [comment]
Renamed one file, which was named by mistake.

**Comment by vitaly-t** [comment]
@flovilmart it is now ready to be merged.

File: src/Adapters/Storage/Postgres/sql/index.js (hunk 1 above)
Comment (vitaly-t) [inline src/Adapters/Storage/Postgres/sql/index.js]: this one was done.

**Comment by vitaly-t** [comment]
Build was successful, please merge it ;)

**Comment by flovilmart** [comment]
Looking awesome thanks!!

**Pull request merged**